lib/HttpClient.php

```diff
@@ -15,36 +15,38 @@ public function __construct($baseUrl, $apiKey, $apiVersion)
         $this->apiVersion = $apiVersion;
     }
 
-    public function get($path, $params = array())
+    public function get($path, $params = [])
     {
         return $this->request($path, 'GET', $params);
     }
 
-    public function delete($path, $params = array())
+    public function delete($path, $params = [])
     {
         return $this->request($path, 'DELETE', $params);
     }
 
-    public function put($path, $body = array(), $params = array())
+    public function put($path, $body = [], $params = [])
     {
         return $this->request($path, 'PUT', $params, $body);
     }
 
-    private function request($path, $httpMethod, $params = array(), $body = null)
+    private function request($path, $httpMethod, $params = [], $body = null)
     {
         $options = array(
             CURLOPT_HTTPHEADER => $this->headers(),
             CURLOPT_CUSTOMREQUEST => $httpMethod,
             CURLOPT_USERPWD => $this->apiKey . ":",
-            CURLOPT_TIMEOUT => $params['timeout'] ?: 5,
+            CURLOPT_TIMEOUT => isset($params['timeout']) ? $params['timeout'] : 5,
             CURLOPT_RETURNTRANSFER => true,
-            CURLOPT_HEADER => 0
+            CURLOPT_FOLLOWLOCATION => true,
```

Comment:
It looks like the `pause` endpoint returns a 302 redirect if the "friendly" user-defined key is used, so this will now follow redirects and execute accordingly.